tests/func_test.py

```diff
@@ -1,5 +1,7 @@
-import sys
 import json
+import sys
+
+from jsonschema.validators import Draft4Validator, create
 
```

Comment:
You can get rid of this newline; third-party imports should be in the same section, so it otherwise looks good.